subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy

```diff
@@ -71,7 +71,7 @@ class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
         executer.inDirectory(file("second")).withTasks("wrapper").run()
 
         then: "the checksum should be constant (unless there are code changes)"
-        sha256(file("first/gradle/wrapper/gradle-wrapper.jar")).asHexString() == "3888c76faa032ea8394b8a54e04ce2227ab1f4be64f65d450f8509fe112d38ce"
+        sha256(file("first/gradle/wrapper/gradle-wrapper.jar")).asHexString() == "96f793a18e056c23ffeec67c1f3bb8eccff5a4a407fc9ceac183527e7eedf4b6"
```

Comment:
So I rebased and I had to undo a couple of the changes as they were test classes where tests were actually relying on them being non-static.

Other than that, this is the only other notable change. Given I've touched the `org.gradle.wrapper.PathAssembler` class, the bytecode is different and so this hash must've changed.